includes/woopay/class-woopay-session.php

```diff
@@ -902,6 +902,7 @@ private static function get_option_fields_status() {
 		$address_2      = get_option( 'woocommerce_checkout_address_2_field', 'optional' );
 		$phone          = get_option( 'woocommerce_checkout_phone_field', 'required' );
 		$terms_checkbox = ! empty( get_option( 'woocommerce_terms_page_id', null ) );
+		$terms_text     = get_option( 'woocommerce_checkout_privacy_policy_text' );
```

Comment:
Different issue, although mentioned in the other comment for this line:

The placeholders aren't being replaced with the current approach. We could use WC's `wc_replace_policy_page_link_placeholders()` for this.

Reply:
Fixed with a52190b7aac1561238c3c355e4bf339630f0caa0